fiftyone/zoo/models/manifest-torch.json

```diff
@@ -8406,6 +8409,49 @@
                 { "name": "MS COCO 2017", "url": "https://cocodataset.org" }
             ],
             "date_added": "2025-07-09 12:00:00"
+        },
+        {
+            "base_name": "hrm2-torch",
+            "base_filename": null,
+            "version": "2.0b",
+            "description": "Reconstructs 3D human body meshes from images using the SMPL body model with state-of-the-art accuracy",
+            "source": "https://github.com/shubham-goel/4D-Humans",
+            "author": "Shubham Goel, et al.",
+            "license": "BSD 3-Clause",
```

Comment:
I see MIT license on the GH repo: https://github.com/shubham-goel/4D-Humans

Reply:
done